Main.java

```diff
@@ -0,0 +1,16 @@
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+public class Main {
+
+	public static void main(String[] args) {
+        // 出力のフォーマット
+        SimpleDateFormat f = new SimpleDateFormat("yyyy/MM/dd HH:mm:sss");
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The SimpleDateFormat object was created without setting the time zone. Make sure that you want to use the default time zone or use letter z, Z or X in the pattern. Otherwise, if you do not call setTimeZone() on the created SimpleDateFormat object, you might get an unexpected date and time when using the object.